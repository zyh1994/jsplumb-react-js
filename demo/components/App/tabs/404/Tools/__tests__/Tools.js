import React from 'react';
import renderer from 'react-test-renderer';
import Tools from '../Tools';
test('404 tools', function () {
    var component = renderer.create(<Tools />);
    var tools = component.toJSON();
    expect(tools).toMatchSnapshot();
});
