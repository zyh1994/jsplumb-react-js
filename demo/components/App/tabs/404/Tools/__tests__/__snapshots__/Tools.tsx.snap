// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 tools 1`] = `
<div
  className="jspr-tools"
>
  <h4>
    404
  </h4>
</div>
`;
